### ISSUE TITLE
Is it possible to support resizing tables in HTML?

### ISSUE THREAD

**Issue by liubinghao123** [TongchengOpenSource/smart-doc#548]
## Summary(关于这个pr的描述)

Brief explanation of the feature.

### Basic example(pr的用例)

If the proposal involves a new or changed API, include a basic code example. Omit this section if it's not applicable.

### Motivation(提出这个pr目的)

Why are we doing this? What use cases does it support? What is the expected outcome?

**Comment by Fioooooooo** [comment]
What do you mean by resizing? Manually adjusting the column width?
指的是如何调整大小，手动调整列宽吗？

**Comment by liubinghao123** [comment]
Yes, currently, the tables take up a lot of space in the layout. One has to scroll a lot to see all the data for an interface.
是的， 目前表格占的排版比较大， 一个接口的数据要滚动很多才看得完

**Comment by liubinghao123** [comment]
Could you provide some parameters to reduce the font size and table size a bit?
是否可以提供一些参数， 可以把字体、表格调小一点

**Comment by shalousun** [comment]
> Could you provide some parameters to reduce the font size and table size a bit?
是否可以提供一些参数， 可以把字体、表格调小一点

Is it because you have a lower screen resolution, or are there too many interface fields? The current sizes given are standard; if the font is too small, it's hard for the eyes to read.
是不是你们屏幕分辨率较低，或者是接口字段过多，现在给的都属于常规大小了，字体这些小了人眼看起来也费劲
